config/secrets.yml.dev

```diff
@@ -21,7 +21,7 @@ secret_keys: &SECRET_KEYS
     web_subscription_id: FILL_THIS_IN_ONCE_WE_HAVE_A_WEB_SUBSCRIPTION_ID
     image_subscription_id: FILL_THIS_IN_ONCE_WE_HAVE_AN_IMAGE_SUBSCRIPTION_ID
   datadog:
-    api_enabled: false
+    api_enabled: true
```

Comment:
Wouldn't we want to keep this as `false` for dev secrets?

Reply:
Nope - see my comments